examples/websites/README.md

```diff
@@ -45,12 +45,12 @@ Published to <your peer id>: /ipfs/$SITE_HASH
 
 Now, you can test that it worked by viewing: `http://localhost:8080/ipns/<your peer id>`.
 And also try the same link on the public gateway. Once you're convinced that works,
-lets again hide the hash. Change your DNS TXT record to `dnslink=/ipns/<your peer id>`,
+lets again hide the hash. Create a DNS TXT record for `_dnslink.your.domain` with `dnslink=/ipns/<your peer id>`,
 wait for that record to propogate, and then try accessing `http://localhost:8080/ipns/your.domain`.
 
 At this point, you have a website on ipfs/ipns, and you may be wondering how you could expose it at `http://your.domain`, so that the Internet users of today may access it too without them having to know about any of this. It's actually surpisingly simple to do, all you need for this is your previously created TXT record and to point the A record of `your.domain` to the ip address of an ipfs daemon that listens on port 80 for HTTP requests (such as `gateway.ipfs.io`). The users' browsers will send `your.domain` in the Host header of the requests, and you have your dnslink TXT records, so the ipfs gateway will recognize `your.domain` as an IPNS name, and so it will serve from under `/ipns/your.domain/` instead of `/`.
 
-So, if you point `your.domain`'s A record to the IP of `gateway.ipfs.io`, and then wait for the DNS to propogate, then anyone should be able to access your ipfs-hosted site without any extra configuration simply at `http://your.domain`.
+So, if you create a CNAME for `your.domain` pointing to `gateway.ipfs.io`, and then wait for the DNS to propogate, then anyone should be able to access your ipfs-hosted site without any extra configuration simply at `http://your.domain`.
```

Comment:
What *are* the tradeoffs, specifically, the disadvantages of `_dnslink` entries over the domain itself?

Giving people multiple options is also a potential source of confusion, especially in examples (where often 'best practises' seem to be demonstrated). Although there is also a clear case for documenting, as reference, the full functionality of the software.